h/badge/views.py

```diff
@@ -0,0 +1,33 @@
+from pyramid import httpexceptions
+
+from h import models
+from h.api.views import api_config
+from h.api.resources import Root
+from h.api import search as search_lib
```

Comment:
Ah. Well here's the thing. I don't think this should live under `/api`. It's not part of our public API. In keeping with where stuff like the session JSON blob lives, I think it should live at `/app/badge` or whatever.

I'm not going to block merging on this but we should change this before we push it out.